src/components/Dropdown/DropdownConstants.ts

```diff
@@ -0,0 +1,35 @@
+interface DropdownObject {
+    id: number;
+    name: string;
+    link: string;
+}
```

Comment:
can we have a different file for this specifically for types like dropdown.types.ts?